### PR TITLE
Label/justin

### PR DESCRIPTION
**Pull request by juztinlazaro**

### CODE REVIEW

File: docs/sections/Docs-Labels/index.jsx [hunk 1]
```diff
@@ -4,7 +4,98 @@ export default class Labels extends React.Component {
...
+			   <div className="add-margin-40x">
+				    <h3>Badges</h3>
+
+			        News Feed  <span className="badges -bordered">20</span>
+
+			        <br/>
+
+			        News Feed <span className="badges -primary">40</span>
+
+			        <br/>
```
Comment (srph) [inline docs/sections/Docs-Labels/index.jsx]: No `<br />`, separate it with `divs`.

``` html
<div>News Feed <span>...</span></div>
```

File: docs/sections/Docs-Labels/index.jsx [hunk 2]
```diff
@@ -4,7 +4,98 @@ export default class Labels extends React.Component {
...
+
+			        News Feed <span className="badges -crimson">80</span>
+			   </div>
+	        </div>
+	    	<div className="col-md-12 add-margin-40x">
+	    		<span className="status -active"></span> Active
+	    		<span className="status -inactive"></span> Inactive
+	    	</div>
+        </div>
```
Comment (srph) [inline docs/sections/Docs-Labels/index.jsx]: Fix indents

File: styles/components/label.scss [hunk 3]
```diff
@@ -19,6 +19,61 @@
...
+.add-margin-40x {
+  margin-top: 50px;
+}
+
+span {
+  margin: 0px 5px 10px;
+}
+
+//place-holder
+%bordered {
```
Comment (srph) [inline styles/components/label.scss]: IMO, this is better:

``` sass
@mixin label-variant($bg, $color: $brand-white) {
  background: $bg;
  color: $color;
}
```

File: styles/components/label.scss [hunk 4]
```diff
@@ -19,6 +19,61 @@
...
+  background-color: $brand-gray;
+  color: $brand-white;
+  border: 1px solid $brand-gray;
```
Comment (srph) [inline styles/components/label.scss]: I don't think we need the border

File: styles/components/label.scss [hunk 5]
```diff
@@ -33,6 +88,35 @@
...
+   &.-primary {
+    @extend %primary;
+   }
+
+   &.-crimson {
+     @extend %crimson;
+   }
+
+   &.-emerald {
+     @extend %emerald;
```
Comment (srph) [inline styles/components/label.scss]: And then,

``` sass
@include label-variant($brand-emerald);
```

File: styles/components/label.scss [hunk 6]
```diff
@@ -88,4 +172,56 @@
...
+     @extend %sky;
+   }
+
+   &.-sun {
+     @extend %sun;
+   }
+
+}
+
+.status {
```
Comment (srph) [inline styles/components/label.scss]: This is for?

**Comment by srph** [comment]
Mind providing a screenshot? :smile: 

**Edit**: Remove screenshot you included in the commit

File: docs/sections/Docs-Labels/index.jsx [hunk 7]
```diff
@@ -4,7 +4,98 @@ export default class Labels extends React.Component {
...
+
+		         <label className="label -sky"> 
+		         	Info 
+		         </label>
+
+		         <label className="label -sun"> 
+		         	Warning 
+		         </label>
+
+		         <label className="label -crimson">
```
Comment (srph) [inline docs/sections/Docs-Labels/index.jsx]: Use `span` instead :smile:.

File: docs/sections/Docs-Labels/index.jsx [hunk 8]
```diff
@@ -4,7 +4,98 @@ export default class Labels extends React.Component {
   render() {
     return (
       <div>
-      </div>
+      	 <h2 className="title">
```
Comment (srph) [inline docs/sections/Docs-Labels/index.jsx]: This is specific to the docs, you can move it to `docs/styles/main.scss` or just leave it alone (aka, delete changes related to doc styling).

File: styles/components/label.scss [hunk 9]
```diff
@@ -19,6 +19,61 @@
 //   font-size: #{$local-font-size}em;
 // }
 
+.title {
```
Comment (srph) [inline styles/components/label.scss]: This is specific to the docs, you can move it to `docs/styles/main.scss` or just leave it alone (aka, delete changes related to doc styling).

File: styles/components/label.scss [hunk 10]
```diff
@@ -19,6 +19,61 @@
...
+.variations-label {
+  > div {
+    > label {
+      margin-right: 5px;
+      margin-top: 5px;    
+    }
+  }
+} 
+
+.add-margin-40x {
```
Comment (srph) [inline styles/components/label.scss]: This is specific to the docs, too.

File: styles/components/label.scss [hunk 11]
```diff
@@ -19,6 +19,61 @@
 //   font-size: #{$local-font-size}em;
 // }
 
+.title {
+    color: $brand-blue;
+    font-size: 30px;
+}
+.variations-label {
```
Comment (srph) [inline styles/components/label.scss]: Specific to the docs. We need this out.

File: styles/components/label.scss [hunk 12]
```diff
@@ -33,6 +88,35 @@
 
   // For changes in modifiers
   transition: 0.15s all;
+  //content
+  
+  //LABELS
```
Comment (srph) [inline styles/components/label.scss]: I think `// modifiers` would be more descriptive

File: styles/components/label.scss [hunk 13]
```diff
@@ -33,6 +88,35 @@
 
   // For changes in modifiers
   transition: 0.15s all;
+  //content
```
Comment (srph) [inline styles/components/label.scss]: ?

File: styles/components/label.scss (hunk 4 above)
Comment (juztinlazaro) [inline styles/components/label.scss]: okay okay, meron kasi sa default na 1px, border: 0px ko nalang no?

File: docs/sections/Docs-Labels/index.jsx (hunk 7 above)
Comment (juztinlazaro) [inline docs/sections/Docs-Labels/index.jsx]: sige haha

File: styles/components/label.scss (hunk 4 above)
Comment (srph) [inline styles/components/label.scss]: Ayyy, my bad, nevermind. I'll work on it. Proceed :smile:

File: styles/components/label.scss (hunk 5 above)
Comment (juztinlazaro) [inline styles/components/label.scss]: saan to nakalagay? :D

File: styles/components/label.scss (hunk 4 above)
Comment (juztinlazaro) [inline styles/components/label.scss]: thanks :D

File: styles/components/label.scss (hunk 12 above)
Comment (juztinlazaro) [inline styles/components/label.scss]: sige :D

File: styles/components/label.scss (hunk 6 above)
Comment (juztinlazaro) [inline styles/components/label.scss]: gumamit ako ng placeholder % para mareuse ko yung property. mixin nalang no?

File: styles/components/label.scss (hunk 3 above)
Comment (juztinlazaro) [inline styles/components/label.scss]: sige mixin ko nalang :D

File: styles/components/label.scss (hunk 6 above)
Comment (juztinlazaro) [inline styles/components/label.scss]: dun sa inactive and active if sa status

Comment (srph) [inline styles/components/label.scss]: Di, yung `.status` hahaha

**Comment by srph** [comment]
Fix code style (indents, etc)..

File: styles/components/label.scss (hunk 6 above)
Comment (juztinlazaro) [inline styles/components/label.scss]: sa inactive and active :D

**Pull request merged**